modules/references/client/components/Info.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+// TODO we can't use the config/config for some reason
+import * as config from '../../../../config/env/default';
+
+/**
+ * @TODO make these elements nicer
+ */
+
+function UserLink({ user }) {
+  return (<strong><a href={`/profile/${user.username}`}>{user.displayName || user.username}</a></strong>);
+}
+
+UserLink.propTypes = {
+  user: PropTypes.object.isRequired
+};
+
+export function Self() {
+  return (<div className="alert alert-warning">Sorry, you can&apos;t give a reference to yourself.</div>);
+}
+
+export function Loading() {
+  return (<div className="alert alert-warning">Loading</div>);
+}
+
+export function Duplicate({ userTo }) {
+  return (<div className="alert alert-warning">You&apos;ve already given a reference to <UserLink user={userTo} />.</div>);
```

Comment:
Translation functions will help with littering `&apos;` in HTML, right? So we’ll be able to:

`{ t( "What's up" ) }`

Reply:
Yes, it will.
I'm not sure about the text within [react-i18next's `<Trans>` component](https://react.i18next.com/components/trans-component), though.